pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -329,6 +350,26 @@ func newRunnerPodForCR(cr *litmuschaosv1alpha1.ChaosEngine, aUUID types.UID, aEx
 				WithArgumentsNew([]string{"-c", "ansible-playbook ./executor/test.yml -i /etc/ansible/hosts; exit 0"}).
 				WithEnvsNew(getChaosRunnerENV(cr, aExList)),
 		).
+		Build()
+	if err != nil {
+		return nil, err
+	}
+	return podObj, nil
+}
+func newExporterPodForCR(cr *litmuschaosv1alpha1.ChaosEngine, aUUID types.UID) (*corev1.Pod, error) {
+	if cr == nil {
+		return nil, errors.New("nil ChaosEngine Object")
+	}
+	labels := map[string]string{
+		"app":  cr.Name,
+		"type": "exporter",
```

Comment:
Please verify do we need to add this `type` label in runner pod also

Reply:
Yes, needed to connect it to the service